longabout.html

```diff
@@ -166,8 +166,8 @@ <h4>MurmurHash3</h4>
 <p>
 Note - The x86 and x64 versions do _not_ produce the same results, as the algorithms are optimized for their respective platforms. You can still compile and run any of them on any platform, but your performance with the non-native version will be less than optimal
 <p>
-<h4>libFFI pre 3.4</h4>
-libffi - Copyright (c) 1996-2021  Anthony Green, Red Hat, Inc and others.
+<h4>libFFI 3.4.2</h4>
+libffi - Copyright (c) 1996-2022  Anthony Green, Red Hat, Inc and others.
```

Comment:
I'm not sure 2022 is correct, 3.4.2 is released in 2021. Are we taking more recent versions than 3.4.2?

Reply:
It might be ambiguous for version but we always take the latest changes from https://github.com/libffi/libffi instead of https://github.com/libffi/libffi/releases/tag/v3.4.2 (2021) which doesn't include the latest changes in RISC-V at https://github.com/eclipse-openj9/openj9/pull/14699 which was merged earlier this year.
```
commit aa3fce08ba620c50db17215a9f14dd0f1facf741
Author: Andreas Schwab [schwab@linux-m68k.org](mailto:schwab@linux-m68k.org)
Date: Sun Feb 13 21:04:33 2022 +0100 <-------
```

Reply:
Then I think we should use `pre 3.5` as the version.

Reply:
Updated to `pre 3.5` as suggested.